.github/workflows/build-antioch.yml

```diff
@@ -0,0 +1,20 @@
+name: build-antioch
```

Comment:
can you check how much this is gonna cost me?

Reply:
Its free

Reply:
https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions

Reply:
pog